stock.py

```diff
@@ -290,13 +291,10 @@ def scan(cls, shipments):
         for shipment in shipments:
             product = shipment.scanned_product
             scanned_quantity = shipment.scanned_quantity
-            if (not product or not scanned_quantity
-                    or shipment.scanned_quantity <= 0):
-                continue
-
-            shipment.process_moves(shipment.get_matching_moves())
-            shipment.clear_scan_values()
-            shipment.save()  # TODO: move to save multiple shipments?
+            if product and (scanned_quantity or scanned_quantity >= 0):
```

Comment:
I think the right condition is:

if product and scanned_quantity and scanned_quantity >= 0

(not the change for "or" into "and")

Reply:
Res a fer. Aquest PR està tancat. Es va abortat i el #7 es el PR de la tasca 047137